api-admin/src/main/resources/application.yml

```diff
@@ -20,6 +20,9 @@ env:
     storageContainerName: ${STORAGE_CONTAINER_NAME:}
     billingProfileId: ${BILLING_PROFILE_ID:550212cf-45dc-478f-9fe6-9b3290e2a1fe}
     deploymentZone: ${DEPLOYMENT_ZONE:}
+  dsm:
+    basePath: ${DSM_ADDRESS:https://dsm-dev.datadonationplatform.org/ddp}
```

Comment:
I've had to think about this for TDR as well: how do we ensure that our prod Juniper instance doesn't send data to dev DSM? It looks like this would silently send data to dev if we forget to explicitly set the dsm address for prod (a potentially easy thing to do)

Reply:
That's a great point. I think it's possible to use `application-development.yml` as an overlay when running locally, but won't be bundled in any artifact that is run in any other environment. I ran a quick experiment and it didn't work the way I wanted it to. Could research this some more, but I'm not going to spend that time right now.